e2e/package.json

```diff
@@ -3,7 +3,7 @@
   "name": "@rsdoctor/e2e",
   "version": "0.0.1",
   "scripts": {
-    "test": "playwright test"
+    "test": "playwright test --update-snapshots"
```

Comment:
should we add a `test:update` script for this?

I'm not sure if this change will make the E2E CI to update snapshot too.

Reply:
This is my mistake, i forgot to delete the test code.